### PR TITLE
docs(ui): fix *SelectValues

### PR DESCRIPTION
**Pull request by kingmigdor**
## Summary

I forgot to edit the docs of *Select.values in #848 , so I edited it.

## Checklist

- [ ] If code changes were made then they have been tested.
  - [ ] I have updated the documentation to reflect the changes.
  - [ ] I have run `task pyright` and fixed the relevant issues.
- [ ] This PR fixes an issue.
- [ ] This PR adds something new (e.g. new method or parameters).
- [ ] This PR is a breaking change (e.g. methods or parameters removed/renamed)
- [x] This PR is **not** a code change (e.g. documentation, README, ...)

### CODE REVIEW

**Review by ooliver1 — COMMENTED** [review]
image

Makes some incredible docs. May be better to remove inherited members and to simply mention the bases.

**Comment by kingmigdor** [comment]
those pyright issues aren't made by me, it's just all the other files because the pyright version changed smh

**Comment by ooliver1** [comment]
No, it is the installation issues.

**Comment by kingmigdor** [comment]
> image
> 
> Makes some incredible docs. May be better to remove inherited members and to simply mention the bases.

I feel like it can be just removed, there's nothing really in the SelectValuesBase that is meant to be used.

File: nextcord/ui/select/mentionable.py [hunk 1]
```diff
@@ -60,12 +60,12 @@ def members(self) -> List[Member]:
 
     @property
     def users(self) -> List[User]:
-        """List[:class:`User`]: A list of users that were selected."""
+        """List[:class:`.User`]: A list of users that were selected."""
```
Comment (ooliver1) [inline nextcord/ui/select/mentionable.py:63]: `Member` needs editing too

**Review by DenverCoder1 — APPROVED** [review]

**Review by ooliver1 — APPROVED** [review]

**Pull request merged**